### ISSUE TITLE
Can not find 'example_scenes.py' when running example on Windows 10 

### ISSUE THREAD

**Issue by stevenschn** [3b1b/manim#956]
I spent about a full day trying to get manim to work. First I had issues with python 3.8, hence I uninstalled and now run 3.7. Then I ran into problems with cairo, but made it work.

Finally I succeeded in installing manimlib, so I thought I made it. 
If I run the following in the prompt, now from the python folder, it seems that `manim` is not correctly detected:

```
C:\Users\steve\AppData\Local\Programs\Python\Python37-32>python -m manim example_scenes.py -pl
C:\Users\steve\AppData\Local\Programs\Python\Python37-32\python.exe: No module named manim
```

But when I give the following command, it does seem to do something:
`python C:\Users\steve\OneDrive\Documenten\Nerds\Manim\manim\manim.py example_scenes.py SquareToCircle -pl` from my python 3.7 folder.

I get the following error:
```
Traceback (most recent call last):
  File "C:\Users\steve\OneDrive\Documenten\Nerds\Manim\manim\manim.py", line 5, in <module>
    manimlib.main()
  File "C:\Users\steve\OneDrive\Documenten\Nerds\Manim\manim\manimlib\__init__.py", line 11, in main
    config = manimlib.config.get_configuration(args)
  File "C:\Users\steve\OneDrive\Documenten\Nerds\Manim\manim\manimlib\config.py", line 185, in get_configuration
    module = get_module(args.file)
  File "C:\Users\steve\OneDrive\Documenten\Nerds\Manim\manim\manimlib\config.py", line 180, in get_module
    spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 859, in get_code
  File "<frozen importlib._bootstrap_external>", line 916, in get_data
FileNotFoundError: [Errno 2] No such file or directory: 'example_scenes.py'
```
example_scenes.py is in the main manim folder. I do not understand this error. Any help would be much appreciated. 

The only thing I can think of that might be related, is the fact I could not install `readlines` by `pip`. This threw an error while installing `manimlib`. It gave an error that the module (being readlines) was not meant for windows. Therefore I installed `pyreadline`, but I never changed any code to for example use `pyreadlines` as `readlines`, since I did not find anything like that anywhere in the code.

PS: I am quite new to github. I try to adhere to all the rules and I searched extensively before asking this question, but I do not know what else to do.

**Comment by kiocosta** [comment]
I guess you probably have already watched it, but first I'd suggest [this tutorial](https://www.youtube.com/watch?v=ZltiKHFWmv8).

If you indeed have watched it and got this error, I believe you should wait a bit more to see if you can get some answers here to get it working, but from my experience, I've tried to use manim in Windows about a year ago and it was a terrible experience. I then changed to Ubuntu and finally got it working.

Unfortunately I don't know how to solve your problem, but if you get stuck for too long there, my suggestion would be to use a dualboot with Ubuntu and install it there. I believe it will be way smoother even though you might still face some issues.

**Comment by stevenschn** [comment]
Thanks for your reply. I will thoroughly watch that tutorial and see if it matches all the steps I took. I was not aware of that tutorial by the way. 

I have no experience with dual boot or with ubuntu. You reckon it is worth the effort if the (at least for now) sole purpose of it is to get manim to work more easily?

**Comment by stevenschn** [comment]
I just did another run on the requirements install by (which I did multiple times before) running from my manim main folder:

`py -m pip install -r requirements.txt`

Note that I need to type `py`, if I use `python` or `python3` it won't work. It then found that both scipy and numpy were from a higher than compatible version, hence it uninstalled those are installed the correct versions.  I then used the following command, again from the manim main folder:

`py manim examples_scenes.py SquareToCircle -pl
`
Which worked. Thank god.
